presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -290,6 +292,16 @@ Name                                        Description
                                             It only applies to ``Insert`` and ``CreateTableAsSelect`` queries.
                                             It would verify each partition's data checksum if the inserted table is partitioned.
                                             It would verify each bucket's data checksum if the inserted table is bucketed.
+``function-substitutes``                    Specification of function substitutions, in the format of
+                                            ``/foo(c0,_)/bar(c0)/,/fred(c0,c1)/baz(qux(c1,c0))/,/foobar(c0)/if(qux(c1),bar(c0),baz(c1))/,...``,
+                                            where ``foo(c0, _)`` would be substituted by ``bar(c0)``, with the declared arguments
+                                            applied to the corresponding positions. Concatenate function substitutions with comma.
```

Comment:
```suggestion
                                            applied to the corresponding positions. 
                                            
                                            Concatenate function substitutions with a comma.
```
Suggest adding line break to left-justify this sentence in the Description column for readability.